src/instructlab/model/evaluate.py

```diff
@@ -79,8 +98,45 @@ def evaluate_model(
             few_shots,
             batch_size,
             tasks_dir,
+            input_questions,
         )
 
+        if benchmark == Benchmark.DK_BENCH:
+            # First Party
+            from instructlab.model.dk_bench_utils import (
+                make_run_dir,
+                print_results,
+                run_dk_bench,
+                validate_output_file_formats,
+                write_results,
+            )
+
+            # turn output_file_formats into a list to be passed into write_results
+            file_formats = output_file_formats.split(",")
+            validate_output_file_formats(file_formats)
+
+            result, model_name = run_dk_bench(
+                ctx,
+                model,
+                max_workers,
+                gpus,
+                backend,
+                enable_serving_output,
+                input_questions,
+                system_prompt,
+                temperature,
+                judge_model,
+            )
+
+            # default for output_dir is set by Click in src/instructlab/cli/model/evaluate.py
+            # it is a string not a pathlib.Path
+            output_dir = make_run_dir(output_dir)
+            files = write_results(result, file_formats, output_dir, model_name)
+            print_results(result, files, model_name)
+
+            print("\n")
+            logger.info("ᕦ(òᴗóˇ)ᕤ Model evaluate with DK-Bench completed! ᕦ(òᴗóˇ)ᕤ")
```

Comment:
The rest of the code is printing (or click echoing) these success messages.  You could make an argument for either but we should be consistent.

Reply:
Ah, I am guessing we are still in the middle of the changes?  Looks like train, sdg, convert, download, and test haven't been switched to use logger yet.

Reply:
@alinaryan Is the reason that the print was changed to be part of the logger for future RESTAPI server integration?